### PR TITLE
New text for sudden events section

### PR DESCRIPTION
**Pull request by bbriscoe**
Intended to resolve NTAP/rfc8312bis/#98

### CODE REVIEW

File: draft-ietf-tcpm-rfc8312bis.md [hunk 1]
```diff
@@ -993,8 +993,14 @@ might be very high after restarting from these periods.
 
 ## Responses to Sudden or Transient Events
 
-If there is a sudden congestion, a routing change, or a mobility
-event, CUBIC behaves the same as AIMD TCP.
+If there is a sudden increase in capacity, e.g. due to variable radio
+capacity, a routing change, or a mobility event, Cubic is designed to
```
Comment (larseggert) [inline draft-ietf-tcpm-rfc8312bis.md:997]:
```suggestion
capacity, a routing change, or a mobility event, CUBIC is designed to
```

File: draft-ietf-tcpm-rfc8312bis.md [hunk 2]
```diff
@@ -993,8 +993,14 @@ might be very high after restarting from these periods.
...
+utilize the newly available capacity faster than Reno.
+
+On the other hand, if there is a sudden decrease in capacity, Cubic
```
Comment (larseggert) [inline draft-ietf-tcpm-rfc8312bis.md:1000]:
```suggestion
On the other hand, if there is a sudden decrease in capacity, CUBIC
```

File: draft-ietf-tcpm-rfc8312bis.md [hunk 3]
```diff
@@ -993,8 +993,14 @@ might be very high after restarting from these periods.
...
+reduces more slowly than Reno. This remains true whether or not Cubic
```
Comment (larseggert) [inline draft-ietf-tcpm-rfc8312bis.md:1001]:
```suggestion
reduces more slowly than Reno. This remains true whether or not CUBIC
```

File: draft-ietf-tcpm-rfc8312bis.md [hunk 4]
```diff
@@ -993,8 +993,14 @@ might be very high after restarting from these periods.
...
+is in Reno-Friendly mode and whether or not fast convergence is
```
Comment (larseggert) [inline draft-ietf-tcpm-rfc8312bis.md:1002]:
```suggestion
is in Reno-friendly mode and whether or not fast convergence is
```

**Review by larseggert — APPROVED** [review]
OK modulo some nits.

**Review by goelvidhi — APPROVED** [review]

**Review by lisongxu — COMMENTED** [review]
Thanks

**Pull request merged**